logic/src/commonMain/kotlin/com/wire/kalium/logic/configuration/GlobalConfigRepository.kt

```diff
@@ -0,0 +1,31 @@
+package com.wire.kalium.logic.configuration
+
+import com.wire.kalium.logic.StorageFailure
+import com.wire.kalium.logic.functional.Either
+import com.wire.kalium.logic.wrapStorageRequest
+import com.wire.kalium.persistence.config.GlobalAppConfigStorage
+import kotlinx.coroutines.flow.Flow
+
+interface GlobalConfigRepository {
+    fun persistEnableLogging(enabled: Boolean): Either<StorageFailure, Unit>
+    fun isLoggingEnabled(): Either<StorageFailure, Boolean>
+    fun isPersistentWebSocketConnectionEnabledFlow(): Flow<Either<StorageFailure, Boolean>>
+    fun persistPersistentWebSocketConnectionStatus(status: Boolean): Either<StorageFailure, Unit>
+}
+
+internal class GlobalConfigDataSource internal constructor(
+    private val globalAppConfigStorage: GlobalAppConfigStorage
+) : GlobalConfigRepository {
+
+    override fun persistEnableLogging(enabled: Boolean): Either<StorageFailure, Unit> =
+        wrapStorageRequest { globalAppConfigStorage.enableLogging(enabled) }
+
+    override fun isLoggingEnabled(): Either<StorageFailure, Boolean> =
+        wrapStorageRequest { globalAppConfigStorage.isLoggingEnables() }
+
+    override fun isPersistentWebSocketConnectionEnabledFlow(): Flow<Either<StorageFailure, Boolean>> =
+        globalAppConfigStorage.isPersistentWebSocketConnectionEnabledFlow().wrapStorageRequest()
+
```

Comment:
To try to achieve this later, https://wearezeta.atlassian.net/browse/AR-2257
Wouldn't be better if we made the persistent web connection flag user specific ?

Reply:
This function is used by `ObservePersistentWebSocketConnectionStatusUseCase` which is in the GlobalScope that is why I moved it to global

Reply:
I would like to keep a change like this out of the scope of this PR since it will require to change stuff in the app